format-specs/geoparquet.md

```diff
@@ -12,11 +12,7 @@ This is version 1.1.0-dev of the GeoParquet specification.  See the [JSON Schema
 
 ## Geometry columns
 
-Geometry columns MUST be stored using the `BYTE_ARRAY` parquet type. They MUST be encoded as [WKB](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary).
```

Comment:
Can we keep the information of the first sentence somewhere? (i.e. that for a WKB encoding, the geometry column MUST be stores using the `BYTE_ARRAY` parquet type)

(you kept the "Implementation note" just below that also mentions `BYTE_ARRAY`, but that is not so specific as the above)